packages/sdk/src/IntrospectionEngine.ts

```diff
@@ -268,6 +269,7 @@ export class IntrospectionEngine {
           const binaryPath = await resolveBinary(BinaryType.introspectionEngine)
           debugRpc('starting introspection engine with binary: ' + binaryPath)
 
+          // TODO Why does the engine need the cwd
```

Comment:
```suggestion
          // about `cwd`
          // If the process is spawned from another directory, all file paths would resolve relative to that instead of the prisma directory
          // note that it isn't something engines specific but just a process spawning thing.
          // Paths resolved in engines code include at least:
          // sqlite database paths
          // ssl certificate paths
```